pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.2.3</Version>
+    <Version>5.2.4-beta</Version>
```

Comment:
I agree we should bump the minor version and we also don't need to add the "-beta" as that comes from somewhere else. So this should just be "5.3.0". 

As for releasing this we will not release this until it is ready and at that point we can do it from the most appropriate release branch.